waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/model/AbstractMetaStore.java

```diff
@@ -38,7 +39,7 @@
   private @NotBlank String remoteMetaStoreUris;
   private @Valid MetastoreTunnel metastoreTunnel;
   private @NotNull AccessControlType accessControlType = AccessControlType.READ_ONLY;
-  private transient @NotNull MetaStoreStatus status = MetaStoreStatus.UNKNOWN;
+  private transient @JsonProperty @NotNull MetaStoreStatus status = MetaStoreStatus.UNKNOWN;
```

Comment:
The actual fix, the rest is test related. Explicitly setting it as a JsonProperty. Following: https://github.com/FasterXML/jackson-databind/issues/1184